docs/cli-existing-project.md

```diff
@@ -63,7 +64,13 @@ pnpm add -D @sphinx-labs/plugins
 ```
 
 
-## 3. Install Sphinx Foundry Library
+## 3. Update Foundry
+Make sure you have the latest version of Foundry installed:
```

Comment:
For the sake of conciseness, I think it's redundant to say "Make sure you have the latest version of Foundry installed" given that the title is "Update Foundry"